src/Compilers/CSharp/Portable/Symbols/Synthesized/GeneratedNames.cs

```diff
@@ -375,6 +378,12 @@ internal static string MakeSynthesizedDelegateName(RefKindVector byRefs, bool re
             return pooledBuilder.ToStringAndFree();
         }
 
+        /// <summary>
+        /// Parses the name of a synthesized delegate out into the things it represents.
+        /// </summary>
+        /// <remarks>
+        /// Logic here should match <see cref="MakeSynthesizedDelegateName" /> below.
```

Comment:
Consider using "above" rather than "below", or perhaps remove "below" from this method and the one above.

```suggestion
        /// Logic here should match <see cref="MakeSynthesizedDelegateName" />.
```

Reply:
lol whoops, thanks :)